screwdriver/store/store.go

```diff
@@ -59,15 +59,15 @@ func (c *client) commandURL() (string, error) {
 }
 
 // New returns Store object
-func New(baseURL string, spec *api.Command, sdToken string) (Store, error) {
+func New(baseURL string, spec *util.CommandSpec, sdToken string) (Store, error) {
 	c, err := newClient(baseURL, spec, sdToken)
```

Comment:
I think we should also update `New` of `store` if you update `New` of `api`:
https://github.com/screwdriver-cd/sd-cmd/pull/12/files#diff-cd10e99cd757d4857182f6d4e88c294cR44

Reply:
```New``` of ```api``` do not have argument of ```*api.Command``` .
I thought it does not need to fix like here on ```api```, but if it's my misunderstanding, please tell me.

Reply:
You deleted `error` return in `api` so you should delete `error` return `store`.

Reply:
Oh, there's no case to be error. I've refactored.